HW03/maksim_manzhai/maksim_manzhai_hw_03_t_03.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+require 'time'
+
+class Homework3
+  def task3(log)
+    lines = log.split(/\n/)
+    lines.select! { |line| event_valid?(line) }.map! { |line| Time.parse(line) }
+
+    return '0' if lines.empty? || lines.size == 1
+
+    duration(lines.compact)
+  end
+
+  private
+
+  def event_valid?(string)
+    string.include?('Calling core with action') ? true : false
```

Comment:
No need to add `true/false` because it's already returns this value
```suggestion
    string.include?('Calling core with action')
```